React/Views/RCTNavItem.m

```diff
@@ -15,6 +15,13 @@ @implementation RCTNavItem
 @synthesize leftButtonItem = _leftButtonItem;
 @synthesize rightButtonItem = _rightButtonItem;
 
+-(id)init {
+  if (self = [super init]) {
```

Comment:
nit: use double (( for assignment inside if, i.e.

  if ((self = [super init])) {